internal/controllers/core/kubernetesapply/reconciler.go

```diff
@@ -36,6 +36,11 @@ import (
 	"github.com/tilt-dev/tilt/pkg/model"
 )
 
+type garbageCollectMeta struct {
```

Comment:
what does 'meta' signify here?

Reply:
maybe it should be called DeleteSpec or smth

Reply:
'meta' signifies I was having a hard time naming this object 😉 

(Renamed to `deleteSpec`, thanks for suggestion)